roamance/src/components/navigation/UserMenu.tsx

```
@@ -0,0 +1,114 @@
+            variant="ghost"
+            size="icon"
+            className={cn(
+              "hidden md:flex rounded-full cursor-pointer transition-all duration-300 relative",
+              "bg-white/10 dark:bg-slate-900/30 hover:bg-white/20 dark:hover:bg-slate-800/50",
+              "backdrop-blur-md shadow-sm border border-white/20 dark:border-slate-800/50",
+              "group"
+            )}
+            title="Your Profile"
+          >
+            <User className="h-[18px] w-[18px] text-primary group-hover:text-primary/90 transition-colors" />
+            <span className="absolute -bottom-1 -right-1 w-2 h-2 bg-green-500 rounded-full border-2 border-background animate-pulse-slow"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)